lib/hammer_cli_katello/content_view_component.rb

```diff
@@ -84,16 +84,16 @@ class ListCommand < HammerCLIKatello::ListCommand
       include OrganizationOptions
       include CompositeContentViewNameResolvable
       output do
-        field :id, _("Id")
+        field :id, _("Component Id")
         field :content_view_name, _("Name")
         field :version, _("Version")
+        field :content_view_id, _("Id")
```

Comment:
This works well. We should probably switch the order to show the content view Id first and component Id in this column instead..Thoughts?

Reply:
Sounds good. Should it also read "Content View Id" instead of "Id"?

Reply:
That would make it clearer too.. 👍🏼